inc/admin/admin.php

```diff
@@ -70,7 +70,9 @@ function rocket_post_row_actions( $actions, $post ) {
 	$url                     = wp_nonce_url( admin_url( 'admin-post.php?action=purge_cache&type=post-' . $post->ID ), 'purge_cache_post-' . $post->ID );
 	$actions['rocket_purge'] = sprintf( '<a href="%s">%s</a>', $url, __( 'Clear this cache', 'rocket' ) );
 
-	return $actions;
+	if ( ! apply_filters( 'rocket_skip_post_row_actions', false, $post ) ) {
+		return $actions;
+	}
```

Comment:
This should be located earlier in the method, after the first conditional, to avoid unnecessary processing if we are skipping.

We should also preserve the final return, as we always want this function to return the `$actions` variable.